test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperTextAreaExtensionsTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Mvc.TestCommon;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core
+{
+    /// <summary>
+    /// Test the TextArea extensions in <see cref="HtmlHelperInputExtensions" /> class.
+    /// </summary>
+    public class HtmlHelperTextAreaExtensionsTest
+    {
+        [Fact]
+        public void TextAreaHelpers_UsesSpecifiedExpression()
```

Comment:
Need a non-`null` value to show that it's used. Check everywhere I didn't already make this comment too.